src/components/Auth/Login/Login.react.js

```diff
@@ -207,7 +207,9 @@ class Login extends Component {
 
 	    if (!state.emailError && !state.passwordError && !state.serverFieldError)
 	    {
-	    	state.validForm = true;
+		if(this.state.password !== '' && this.state.email !== ''){
```

Comment:
How about you add this condition to the above condition?
Like this - 
```
if (!state.emailError && !state.passwordError && !state.serverFieldError && this.state.password !== '' && this.state.email !== '')
```

Reply:
Done @DNS-404